### PR TITLE
removing addDefaults from the PipelineJobBuilder

### PR DESCRIPTION
**Pull request by Josh-thephillipsequation**

### CODE REVIEW

**Comment by JimRamia** [comment]
This looks good.  Don't see a merge option though for me.  Someone else will ahve to merge.

File: src/main/groovy/jenkins/automation/builders/PipelineJobBuilder.groovy [hunk 1]
```diff
@@ -32,7 +32,9 @@ class PipelineJobBuilder {
     Job build(DslFactory factory) {
         factory.pipelineJob(name) {
             it.description this.description
-            CommonUtils.addDefaults(delegate)
+            logRotator {
```
Comment (imuchnik) [inline src/main/groovy/jenkins/automation/builders/PipelineJobBuilder.groovy:35]: Why was this line changed?

Comment (Josh-thephillipsequation) [reply src/main/groovy/jenkins/automation/builders/PipelineJobBuilder.groovy:35]: This resolves issue #140 (basically parts of the addDefaults method were deprecated in the new version of the job DSL plugin

Comment (imuchnik) [reply src/main/groovy/jenkins/automation/builders/PipelineJobBuilder.groovy:35]: https://github.com/cfpb/jenkins-automation/blob/master/src/main/groovy/jenkins/automation/utils/CommonUtils.groovy#L14  We should fix the method. This is mostly for uniformity, so we can change that logRotator value if we need to.

**Review by imuchnik — CHANGES_REQUESTED** [review]
We need to fix it in utils, so that it works everywhere it is used

**Comment by imuchnik** [comment]
I think this PR needs to be closed.

File: src/main/groovy/jenkins/automation/builders/PipelineJobBuilder.groovy (hunk 1 above)
Comment (Josh-thephillipsequation) [reply src/main/groovy/jenkins/automation/builders/PipelineJobBuilder.groovy:35]: this was the change requested by @marcesher as the common utils method didn't have any additional non deprecated features beyond the logRotator

**Review by imuchnik — APPROVED** [review]
